### DISCUSSION TITLE
With current backends there is no support for Linux AMD gpu users?

### DISCUSSION THREAD

**Discussion by trsh** [LuisaGroup/LuisaCompute#110] [Q&A] [answered]
With current backends there is no support for Linux AMD gpu users?

**Comment by Mike-Leo-Smith** [comment, marked as answer]
Currently, no. But we may implement HIP or Vulkan-based backends to support AMD on Linux in the future.

**Reply by trsh** [reply]
Yes, please both :)

**Reply by trsh** [reply]
Tho, I think Vulkan is most important
